### PR TITLE
Changing the "versions" string to "version" for Proxy Service

### PR DESCRIPTION
**Pull request by nazliyanik**
## Description
The version string for the proxy service has been written as 'versions'; it needs to be changed.

## Related Issue

- Fixes <issue_link>
[OCSRE-273 Jira Issue](https://kiteworks.atlassian.net/jira/software/c/projects/OCSRE/issues/OCSRE-273?filter=myopenissues&jql=project%20%3D%20%22OCSRE%22%20AND%20assignee%20IN%20%28currentUser%28%29%29%20AND%20statusCategory%20IN%20%28%22To%20Do%22%2C%20%22In%20Progress%

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
Could you provide a changelog @nazliyanik ?

**Comment by nazliyanik** [comment]
> Could you provide a changelog @nazliyanik ?

Provided

File: changelog/unreleased/fix-proxy-build-info.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Bugfix: Fixed proxy build info
+
+The version string for the proxy service has been changed to 'version'.
+
+# 10039
```
Comment (micbar) [inline changelog/unreleased/fix-proxy-build-info.md:5]:
```suggestion
https://github.com/owncloud/ocis/pull/10039
```

**Comment by micbar** [comment]
@nazliyanik I put the correct url into the changelog. Could you approve my suggesttion? Then we can merge this.

Thank you!

File: changelog/unreleased/fix-proxy-build-info.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Bugfix: Fixed proxy build info
+
+The version string for the proxy service has been changed to 'version'.
+
+[10035](https://github.com/owncloud/ocis/pull/10039)
```
Comment (micbar) [inline changelog/unreleased/fix-proxy-build-info.md:5]:
```suggestion
https://github.com/owncloud/ocis/pull/10039
```

Comment (micbar) [reply changelog/unreleased/fix-proxy-build-info.md:5]: This needs to be just an url, not a hyperlink.

**Review by micbar — APPROVED** [review]

**Review by butonic — APPROVED** [review]

**Pull request merged**